tests/e2e/thirdparty.playwright.ts-snapshots/thirdparty-esbuild-next--development.css

```diff
@@ -1,11 +1,11 @@
-.styles_depBlock__vg1esd1 {
-  --depColor__vg1esd0: green;
-  background-color: var(--depColor__vg1esd0);
-}
 .styles_depdepBlock__7q0fkw1 {
   --depdepColor__7q0fkw0: blue;
   background-color: var(--depdepColor__7q0fkw0);
 }
+.styles_depBlock__vg1esd1 {
+  --depColor__vg1esd0: green;
+  background-color: var(--depColor__vg1esd0);
+}
```

Comment:
I don't think this ordering is correct. Looking at the import order of styles in the `thirdparty` fixture (see [here](https://github.com/vanilla-extract-css/vanilla-extract/blob/9a0c6898e02ffe54c8455110ac0a7adecb334acc/fixtures/thirdparty/thirdparty-dep/index.mjs)), I would expect the `depBlock` style to be declared before the `depdepBlock` style.

Reply:
I think I agree. I'll have a look at it to figure out why the order changed here.